File: beginner/list.js
// Lista létrehozása
let gyumolcsok = ["alma", "banán", "cseresznye"];

// Elem hozzáadása
gyumolcsok.push("szőlő");

// Elem törlése
gyumolcsok = gyumolcsok.filter(item => item !== "banán");

// Lista bejárása
gyumolcsok.forEach(gyumolcs => {
  console.log("Gyümölcs:", gyumolcs);
});

// Keresés
if (gyumolcsok.includes("alma")) {
  console.log("Az 'alma' szerepel a listában.");
}

File: beginner/list.py
# Lista létrehozása
gyumolcsok = ["alma", "banán", "cseresznye"]

# Elem hozzáadása
gyumolcsok.append("szőlő")

# Elem törlése
gyumolcsok.remove("banán")

# Lista bejárása
for gyumolcs in gyumolcsok:
    print("Gyümölcs:", gyumolcs)

# Keresés
if "alma" in gyumolcsok:
    print("Az 'alma' szerepel a listában.")
